collector/shared/version.go

```diff
@@ -1,42 +0,0 @@
-package collector_shared
```

Comment:
That is a good point, we may need it later. Let me add this back somewhere for future-proofing.